nuxdie/blog
# Update dependency moment to version 2.18.1

This Pull Request updates dependency moment from version `2.17.1` to `2.18.1`

### Changelog

#### 2.18.1 / 2017-03-21

  * Build 2.18.1
  * Bump version to 2.18.1
  * Update changelog for 2.18.1
  * Merge pull request [#3853](https://github.com/moment/moment/issues/3853) from shmendo:develop
    [misc] Fix invalid whitespace character causing inability to parse moment.js
  * Fixed issue with a whitespace character which caused an error in the browser during parsing in Chrome.

#### 2.18.0 / 2017-03-18

  * Build 2.18.0
  * Bump version to 2.18.0
  * Update changelog for 2.18.0
  * Revert &quot;Merge pull request [#3666](https://github.com/moment/moment/issues/3666) from josephjaniga:calendar-overload-formats&quot;
    This reverts commit a2df2a309ae75d5a2d5ee44666261d8c8a92f804, reversing
    changes made to a52400ec443c942a71584f3bdafddd2847423862.
  * Revert &quot;Merge pull request [#3786](https://github.com/moment/moment/issues/3786) from netguru:develop&quot;
    This reverts commit b1862fd5041e7bbdaf6f62b159e0944ab21e3dc0, reversing
    changes made to a3ab61646cf3c4e5a41a75d0843b26755ca4498a.
  * Merge pull request [#3847](https://github.com/moment/moment/issues/3847) from moment:fix/3631
    [misc] Fix custom locale bundler, fixes [#3631](https://github.com/moment/moment/issues/3631)
  * Fix lint errors
  * Filter out en from locales
  * Merge pull request [#3840](https://github.com/moment/moment/issues/3840) from marwahaha:fix/windows-dev
    [misc] Use npm to run typescript tests, fixes [#3826](https://github.com/moment/moment/issues/3826)
  * Use npm to run typescript tests
  * Merge pull request [#3828](https://github.com/moment/moment/issues/3828) from moment/meteor-readme-simplification
    Simplify Meteor Readme
  * Simplify Meteor Readme
    The Meteor readme needn&#x27;t be an ad for Meteor.
  * Merge pull request [#3793](https://github.com/moment/moment/issues/3793) from fbonzon:ordinal-parse
    [misc] Change ordinalParse to dayOfMonthOrdinalParse in locales
  * Fixup new locales
  * Add fallback for lenient parsing
  * Add fallback for old name ordinalParse
  * Sync with develop branch
  * Rename ordinalParse → dayOfMonthOrdinalParse
    &quot;ordinalParse&quot; was poorly named, because, contrary to &quot;ordinal&quot; short
    which is responsible to display ordinal for many units, the former
    cares only about day of month parsing.
  * Merge pull request [#3819](https://github.com/moment/moment/issues/3819) from sschueller:develop
    [new locale] de_ch: German (Switzerland) locale
  * Update de-ch header
  * Added de_CH (German Switzerland)
  * Added de_CH (German Switzerland) Test
  * Added de_CH (German Switzerland) according to Weisungen der Bundeskanzlei
  * Merge pull request [#3800](https://github.com/moment/moment/issues/3800) from lashad:patch-2
    [locale] ka: Improved past relative time
  * change the tests to meet grammar issue in ka.js file.
  * Update ka.js
    Hello,
    Small grammar error fix.
  * Merge pull request [#3790](https://github.com/moment/moment/issues/3790) from fbonzon:fr-ordinal
    [locale] fr-*: Better ordinal parsing and &#x60; in place of &#x27;
  * Align code formatting of fr-ch, fr-ca and fr locales
    Now a diff between any 2 files of fr-ch, fr-ca and fr locales shows
    actual differences in time formatting rules, not differences in how the
    code is written.
  * Honor grammatical gender for ordinals in French locale
    Other change: French uses curly quote ( ’ ) not straight quote ( &#x27; )
  * Merge pull request [#3708](https://github.com/moment/moment/issues/3708) from TracyGJG:develop
    [feature] RFC2822 parsing
  * Further correction
  * Zulu conversion correction.
  * Testcases revised to track down issue.
  * Correction to military case insensitive time zone
  * Enhanced positive test cases to include a variety of time zone/offsets.
    Made the military time zone case insensitive in line with specification.
  * Changed parsingFlags().rfc2822 from a function call
  * Checking rfc2822 parsingFlags
  * Missing semi-colons added.
  * Positive and Negative RFC2822 testCases streamlined.
  * rfc2822 ParsingFlags moved to the bottom of the parsing code.
  * Update from-string.js
    Issue with configFromString()
  * Update from-string.js
    Trailing white-space removed
  * Update from-string.js
    Semi-colons
  * Update from-string.js
    configFromString() made linear, Military variable made lowercase and RegExp comment replaced with tools.ietf reference.
  * Update parsing-flags.js
    Spacing of the rfc2822 parse flag syntax aligned.
  * Update relative_time.js
    Test case re-enabled.
  * Revert &quot;Further style correction.&quot;
    This reverts commit a9cdeeff05c301f34570d79453e3d4bc55fa0455.
  * Further style correction.
  * Style correction.
  * Parsing flag weekdayMismatch added and applied to the RFC parse validation.
  * Removed the breaking comments in the test case.
  * Correction: Style of comment!
  * Retest rfc2822
  * Trial day-of-week validation using a vanilla Date object.
  * Removed import of moment library.
  * Revoked weekday check and test
  * Style correction
  * Further trial
  * Trial validation of weekday
  * Removed rfc2822 prefix from internal variables.
  * Test case &#x27;RFC2822 datetime with CFWSs&#x27; revised to replace the template literal with a complex string.
  * Revised RFC2822 regular expressions
  * Near-candidate solution
  * Timezone/offset processing added.
  * Work in Progress - Do Not Merge
  * Merge pull request [#3671](https://github.com/moment/moment/issues/3671) from marcoil:locale_ca_add_de
    [locale] ca: Improved months and datetime formats
  * Improved Catalan localization
    * Changed months abbreviations to the recommended ones
    (https://www.upf.edu/leupf/12ap1/111_3.htm).
    * Added articles and &quot;de/d&#x27;&quot; to long-form dates.
    * Added commas to short-form dates.
  * Merge pull request [#3814](https://github.com/moment/moment/issues/3814) from dimkinv:develop
    [misc] typescript: Moment interface now extends Object to inherit toString()
  * Moment interface now extends Object to inherit toString() function
  * Merge pull request [#3812](https://github.com/moment/moment/issues/3812) from webbiesdk:ts-fixes
    [misc] typescript: Minor fixes
  * Small fixes to the .d.ts file
  * Merge pull request [#3817](https://github.com/moment/moment/issues/3817) from billschaller:patch-2
    [misc] Don&#x27;t enumerate momentProperties with &#x27;in&#x27;
  * [Easy] Don&#x27;t enumerate momentProperties with &#x27;in&#x27;
    Because of the way many polyfill libraries modify Array.prototype (without Object.defineProperty), enumerating momentProperties with for...in is unsafe.
    This is a zero-cost change that makes this enumeration safe.
  * Merge pull request [#3815](https://github.com/moment/moment/issues/3815) from marwahaha:fix/3702
    [misc] typescrpt: Fixes [#3702](https://github.com/moment/moment/issues/3702), add week in duration input spec
  * Add a test
  * Add typings for weeks on DurationInputObject
  * Merge pull request [#3806](https://github.com/moment/moment/issues/3806) from timfish:develop
    [bugfix] Fixes [#3805](https://github.com/moment/moment/issues/3805), fix locale month getters for standalone/format cases
  * code style
  * revert two missed files
  * @ichernev tips
  * I should probably run the code style tests locally...
  * get tests passing
  * another code style issue
  * Fix code style errors
  * Add localeData tests
  * Fix issue [#3805](https://github.com/moment/moment/issues/3805)
  * Merge pull request [#3792](https://github.com/moment/moment/issues/3792) from nusretparlak:develop
    [new locale] ar-kw: Arabic (Kuwait) locale
  * Remove locale/ar-kw.js (autogen later)
  * Ar-KW Test Updated
    Ar-KW Test Updated
  * Ar-Kw(Kuwait) Language Test Package Added . 
    Ar-Kw(Kuwait) Language Test Package Added .
